cloud/scaling-a-service.md

```diff
@@ -101,7 +80,13 @@ very quickly. However, if you have a lot of existing data, optimization can take
 longer, and in some cases this could create a delay longer than six hours.
 
 Autoscaling can only increase disk size, not decrease it. You can have a disk up
-to 10&nbsp;TB in size.
+to 16&nbsp;TB in size.
+
+If you have a Timescale Cloud multi-node environment, you can also use
```

Comment:
We usually refer to them as multi-node clusters. But if it's the same across the whole documentation, then I'm ok with environment.

```suggestion
If you have a Timescale Cloud multi-node cluster, you can also use
```